feature-set/src/lib.rs

```diff
@@ -1024,6 +1024,10 @@ pub mod raise_block_limits_to_60m {
     solana_pubkey::declare_id!("6oMCUgfY6BzZ6jwB681J6ju5Bh6CjVXbd7NeWYqiXBSu");
 }
 
+pub mod rent_epoch_is_a_constant_in_vm {
```

Comment:
I would name it an action like "mask out rent epoch in vm serialization" rather than a statement.